.travis.yml

```diff
@@ -11,7 +11,12 @@ jobs:
     - name: “Python 3.8 make lint only”
       python: "3.8"
       install: pip install flake8
-      script: make lint
+      script:
+        - make lint
+        # Run all flake8 tests on modified code
+        - if [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
```

Comment:
Out of curiosity, why is this needed?

Reply:
`origin/master` is not defined in a push...
https://travis-ci.org/internetarchive/openlibrary/jobs/649813381#L354

Reply:
Hmm, wait, then when is it running? Cause it should ideally be run every time one pushes to a pull request.

Reply:
Ahhh, ok; maybe let's add a comment here about why the check exists :+1: